crates/trigger/Cargo.toml

```diff
@@ -8,6 +8,12 @@ edition = { workspace = true }
 llm = ["spin-llm-local"]
 llm-metal = ["llm", "spin-llm-local/metal"]
 llm-cublas = ["llm", "spin-llm-local/cublas"]
+# Enables creating a TriggerLoader that loads AOT compiled components. It
+# enables using the unsafe `wasmtime::deserialize` function, which can be unsafe
```

Comment:
Nit: a `wasmtime::deserialize` function doesn't exist. 

Also, I think we shouldn't necessarily mention the wasmtime function used here and instead point users to `TriggerLoader::new_aot_compiled` where we spell out all of the safety concerns.

Reply:
Pointed to the function docs